2fa/README.md

```diff
@@ -2,8 +2,46 @@
 
 ## What is it?
 
+Two factor authentication (2fa from now on) is a method to double check that you're really you by requiring one more check with something you have. This is usually a code generated by an authenticator application or a physical device, or a text message that's sent to your phone.
+
+When you set up 2fa you provide a special code (seed) of your account / device which will instruct the authenticator to generate the correct codes. This usually happens by scanning a QR code, or by providing the site with a unique identifier of your physical device.
```

Comment:
Are these two paragraphs a bit technical? Full marks for explaining them in clear language, but we might be diving a bit deep here!